climpred/reference.py

```diff
@@ -0,0 +1,212 @@
+import inspect
+
+import xarray as xr
+
+from .alignment import return_inits_and_verif_dates
+from .checks import has_valid_lead_units, is_xarray
+from .comparisons import COMPARISON_ALIASES, HINDCAST_COMPARISONS, __e2c
+from .constants import CLIMPRED_DIMS
+from .metrics import DETERMINISTIC_HINDCAST_METRICS, METRIC_ALIASES
+from .utils import (
+    assign_attrs,
+    convert_to_cftime_index,
+    copy_coords_from_to,
+    get_comparison_class,
+    get_lead_cftime_shift_args,
+    get_metric_class,
+    shift_cftime_index,
+)
+
+
+def persistence(verif, inits, verif_dates, lead):
+    a = verif.where(verif.time.isin(inits[lead]), drop=True)
+    b = verif.sel(time=verif_dates[lead])
+    return a, b
+
+
+def historical(hist, verif, verif_dates, lead):
+    a = hist.sel(time=verif_dates[lead])
+    b = verif.sel(time=verif_dates[lead])
+    return a, b
+
```

Comment:
it would be great to just have `historial`, `perisistence` and `climatology` destilled into such small calls. but this seems rather the time matching only.